### ISSUE TITLE
Fields progguidhtml and meetinglink missing from database creation scripts.

### ISSUE THREAD

**Issue by lostcarpark** [olszowka/Zambia#97]
I set up an instance of Zambia using the SampleDbase.dump script. I then tried to add a session, and received a missing field error.
To get it working, I had to run the following SQL:

```
alter table Sessions add progguidhtml text after pocketprogtext;
alter table Sessions add meetinglink text after progguidhtml;
```
I wasn't sure of the type, but text seems to work.
I expect it should be a simple matter to add these to the install scripts.

**Comment by sydwhv** [comment]
It looks like some of the code from the branch html_bio either leaked to master, or html_bio wasn't fully integrated, those fields are created in 56_html_bio.sql

**Comment by olszowka** [comment]
Fixed with PR linked above.  The schema change above is premature and will cause problems in the future.  At some point that schema change will be made, but the script to do it might have other changes.

**Comment by olszowka** [comment]
I merged a fix to this issue.  The schema change is premature but will
happen eventually.

Are you just trying out Zambia or do you intend to use it for a particular
con?  I like to keep track of which cons are using it.

  --PeterO

**Comment by lostcarpark** [comment]
Thank you. I've pulled the latest codebase, and dropped the extra fields, and confirmed I can add a session.

I'm currently working with the programme team for this year's Eastercon, ConFusion. As this is just a few weeks away, most of the programme work has been completed with spreadsheets, but we expect Zambia to be helpful for sanity checking and preparing the programme presentation. We also plan to integrate with KonOpas for online programme presentation (I've worked with KonOpas before).

I'm involved with a couple of other conventions that are considering Zambia, and would be using it more fully. I'll let you know about those in due course.

**Comment by lostcarpark** [comment]
Oh, one other question, is schema.html maintained? There are a fwe tables that seem to be out of date.

**Comment by olszowka** [comment]
No, it is not being maintained. I no longer have access to the tool that
generated it.

  —PeterO

**Comment by lostcarpark** [comment]
That's okay. I'll know not to rely on it.

**Comment by olszowka** [comment]
Fixed by PR #98
